packages/core-p2p/src/socket-server/utils/get-peer-config.ts

```diff
@@ -25,6 +25,10 @@ const transformPlugins = (plugins: PluginConfig[]): Contracts.P2P.PeerPlugins =>
             enabled: true, // default to true because "enabled" flag is in different place based on which plugin
             port,
         };
+
+        if (name.includes("core-api")) {
+            result[name].estimateTotalCount = pluginConfig.options?.options?.estimateTotalCount || true;
```

Comment:
Why would we need the `|| true` since `estimateTotalCount` is always present from the defaults?

Reply:
If you take a look in packages/core-api/src/controllers/controller.ts it states this is an optional property. 

`
     estimateTotalCount: this.apiConfiguration.getOptional<boolean>("options.estimateTotalCount", true),
`

Reply:
Yes, but that doesn't mean that it is correct. If you look at the `defaults.ts` file you will see that it's always present from the defaults because we merge the user and default configuration so unless someone modifies the defaults and rebuilds core from source I can't think of a situation where this would not be specified.